src/install.rst

```diff
@@ -6,342 +6,267 @@ Installing Nextstrain
-
-         b. Open the downloaded file and follow through installation prompts.
-
-      2. Open a terminal window.
-      3. Install Mamba on the ``base`` Conda environment:
-
-         .. code-block:: bash
-
-            conda install -n base -c conda-forge mamba --yes
-            conda activate base
-
-      4. Install the Nextstrain components. There are two options:
-
-         .. tabs::
-
-            .. group-tab:: Docker (recommended)
-
-               .. warning::
-
-                  If using a newer Mac with an `Apple silicon chip <https://support.apple.com/en-us/HT211814>`_ (e.g. M1), **Native** installation is recommended due to slowness with the Docker installation. `We are considering ways to improve this <https://github.com/nextstrain/docker-base/issues/35>`_.
+Steps vary by runtime option (Docker, native) and operating system (macOS, Windows, WSL on Windows, Linux).
+For help choosing, refer to our :doc:`/reference/faq`, such as:
 
-               1. `Install Docker Desktop using the official guide <https://docs.docker.com/desktop/install/mac-install/>`_.
-               2. Create a Conda environment named ``nextstrain``:
+  * :ref:`what-are-docker-conda-mamba-wsl-etc`
+  * :ref:`choosing-a-runtime`
+  * :ref:`when-to-use-wsl`
 
-                  .. include:: snippets/conda-create-bash.rst
+First, install the Nextstrain components.
 
-               3. Install the Nextstrain CLI:
-
-                  .. code-block:: bash
-
-                     mamba install --yes nextstrain-cli
-
-            .. group-tab:: Native
-
-               1. Create a Conda environment named ``nextstrain``:
-
-                  .. include:: snippets/conda-create-bash.rst
-
-               2. Install all the necessary software:
-
-                  .. include:: snippets/conda-install-full-bash.rst
+.. tabs::
 
-      5. Confirm that the installation worked.
+   .. group-tab:: Docker (recommended)
 
-         .. code-block:: bash
+      .. tabs::
 
-            nextstrain check-setup --set-default
+         .. group-tab:: macOS
 
-         The final output from the last command should look like this, where ``<option>`` is the option chosen in the previous step:
+            .. warning::
 
-         .. code-block:: none
+               If using a newer Mac with an `Apple silicon chip <https://support.apple.com/en-us/HT211814>`_ (e.g. M1), **Native** installation is recommended due to slowness with the Docker installation. `We are considering ways to improve this <https://github.com/nextstrain/docker-base/issues/35>`_.
 
-            Setting default environment to <option>.
+            1. `Install Docker Desktop using the official guide <https://docs.docker.com/desktop/install/mac-install/>`_.
+            2. Install the Nextstrain CLI:
 
-   .. group-tab:: Windows
+               .. code-block:: bash
 
-      .. note::
+                  curl -fsSL --proto '=https' https://nextstrain.org/cli/installer/mac | bash
```

Comment:
With an installation script that does things automatically, it would be good to include instructions on how to uninstall. However, it can get just as complex as the installation docs itself with the various installation paths (current and past). Maybe something along these lines in the FAQ page would be a good start:

1. If `~/.local/share/nextstrain`/`~\AppData\Local\nextstrain` exists, remove it.
2. If everything is in a Conda environment, remove the Conda environment.
3. If Docker was used, remove `nextstrain/*` Docker images.
4. Optionally, uninstall Docker/Conda/WSL if only used for Nextstrain.

I don't think we should worry about installations on an independently managed environment (e.g. `pip install nextstrain-cli`).

In the longer term, maybe a `nextstrain uninstall` command would be better.

Reply:
Ah, good point. I agree. Although instead of an FAQ entry, I'd think to make an "Uninstalling" section at the bottom of the install page, where it'll be more visible and not require a link out from install to FAQ.

Reply:
Pushed as [77424e3](https://github.com/nextstrain/docs.nextstrain.org/pull/123/commits/77424e399b64a016c140009b7594da99b29ff11e).